app/assets/javascripts/pieces.js

```diff
@@ -2,5 +2,11 @@ $(document).ready(function(){
   $("td").click(function(){
```

Comment:
@ilyakrasnov OK, I'll take it out for the moment. I take your point about keeping the code as clean as possible at any given moment. ;)